docs/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebarSortOrder: 0
+sidebarSortOrder: 1000
```

Comment:
why are you trying to put the docs index at the bottom?

Reply:
based on vercel data, it looks like currently the most viewed routes under docs are:

/cluster
/wallet
/rpc
/terminology

rpc and terminology are probably so high up because its the first thing on the top of the page.
it doesn't really make sense for these pages to be so high up as they're primariliy references


eventually, I was thinking of redoing the Intro/Quickstart to starts with an overview of how to use Solana Playground

Reply:
We will adjust this one later with the updates in the intro section that jacob is working on (and the secondary nav). But for now, let's keep it as.

- RPC is one of the most searched sections since those pages are well indexed and linked on the internet
- the terms page is heavily trafficed since it is linked all over, so as people browse the docs or guides, they often click through to get the definition of a given word
- wallet - this one is a mystery.... we suspect high search intent and domain authority. e.g. "solana wallet" sort of makes sense to show a page on solana-com that is about wallets